src/App.tsx

```diff
@@ -1,14 +1,77 @@
-/* eslint-disable max-len */
-import React from 'react';
-import 'bulma/css/bulma.css';
-import '@fortawesome/fontawesome-free/css/all.css';
-
+import React, { useEffect } from 'react';
+import { useAppDispatch, useAppSelector } from './app/hooks';
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { todosActions } from './features/todos';
+import { filterActions } from './features/filter';
+import { currentTodoActions } from './features/currentTodo';
+import { Status } from './types/Status';
+import { Todo } from './types/Todo';
+import 'bulma/css/bulma.css';
+import '@fortawesome/fontawesome-free/css/all.css';
+
+const getLowerString = (str: string) => {
+  return str.toLowerCase();
+};
 
 export const App: React.FC = () => {
+  const todos = useAppSelector(state => state.todos);
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const { status, query } = useAppSelector(state => state.filter);
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    getTodos()
+      .then((todosFromData) => dispatch(todosActions.get(todosFromData)));
+  }, []);
+
+  const handleChangeStatus = (value: string) => {
+    dispatch(filterActions.changeStatus(value as Status));
+  };
+
+  const handleSetQuery = (value: string) => {
+    dispatch(filterActions.setQuery(value));
+  };
+
+  const handleClearQuery = () => {
+    dispatch(filterActions.clearQuery());
+  };
```

Comment:
What do you think about to move it directly in `<TodoFilter />`?

Looks like it can works and can simplify your App component